linux/ibus-keyman/src/keymanutil.c

```diff
@@ -369,6 +368,7 @@ keyman_get_options_queue_fromdconf(gchar *package_id,
             }
             index++;
         }
+        g_strfreev(options);
```

Comment:
I note on line 352 you have: 

```
    if (*options != NULL)
```

But `keyman_get_options_fromdconf` can return `NULL` so we should be testing for that, not `*options` surely.

Reply:
Thanks for noticing. Done.